Dart/testSrc/com/jetbrains/dart/analysisServer/DartServerCompletionTest.java

```diff
@@ -188,4 +188,15 @@ public void testNoCompletionAfterDigit() {
   public void testNotYetImportedClass() {
     doTest("Process");
   }
+
+  public void testExistingImports() {
```

Comment:
This test doesn't fail for me with the new code commented out or against older SDKs. Apparently, it doesn't test the feature.

Reply:
Good point. We really need to be checking that one (and only one) library offers the completion (the already-imported one). I went ahead and updated the test.